<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>311</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>PCSP</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pcsp.qrc">
    <normaloff>:/../../Resources/Resources/pcsp.ico</normaloff>:/../../Resources/Resources/pcsp.ico</iconset>
  </property>
  <widget class="QPspScreen" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_UMD"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_MemCard"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEmulator">
    <property name="title">
     <string>&amp;Emulator</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
   </widget>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionConfigure_Controller"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionVSYNC_Limiter"/>
    <addaction name="actionUse_OPENGL_Shaders"/>
    <addaction name="actionMute_Sound"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionAlwaysOnTop"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="actionOpen_Debugger"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmulator"/>
   <addaction name="menuConfigure"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/open_file.png</normaloff>:/../../Resources/Resources/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/quit.png</normaloff>:/../../Resources/Resources/quit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/play.png</normaloff>:/../../Resources/Resources/play.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/pause.png</normaloff>:/../../Resources/Resources/pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/reset.png</normaloff>:/../../Resources/Resources/reset.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;set</string>
   </property>
  </action>
  <action name="actionVSYNC_Limiter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;VSYNC Limiter</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionConfigure_Controller">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/psp.png</normaloff>:/../../Resources/Resources/psp.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure Controller</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionOpen_MemCard">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/memorycard.png</normaloff>:/../../Resources/Resources/memorycard.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;MemoryCard</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/pcsp.ico</normaloff>:/../../Resources/Resources/pcsp.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>Meta+F12</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen_UMD">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/umd.png</normaloff>:/../../Resources/Resources/umd.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;UMD</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionOpen_Debugger">
   <property name="text">
    <string>Open Debugger</string>
   </property>
  </action>
  <action name="actionAlwaysOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always On Top</string>
   </property>
   <property name="shortcut">
    <string>Meta+F5</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Meta+F3</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="pcsp.qrc">
     <normaloff>:/../../Resources/Resources/screenshot.png</normaloff>:/../../Resources/Resources/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionUse_OPENGL_Shaders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use OpenGL Shaders</string>
   </property>
   <property name="shortcut">
    <string>Meta+F4</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionMute_Sound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mute Sound</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPspScreen</class>
   <extends>QWidget</extends>
   <header location="global">qpspscreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pcsp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRun</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPause</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVSYNC_Limiter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuVSYNC()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConfigure_Controller</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuConfigureController()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_MemCard</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuOpenMemCard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_UMD</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuOpenUMD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Debugger</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuOpenDebugger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAlwaysOnTop</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuAlwaysOnTop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScreenshot</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuScreenshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUse_OPENGL_Shaders</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuUseShaders()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMute_Sound</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuMuteSound()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFullscreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuFullscreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menuOpen()</slot>
  <slot>menuExit()</slot>
  <slot>menuEnableDebugger()</slot>
  <slot>menuRun()</slot>
  <slot>menuPause()</slot>
  <slot>menuReset()</slot>
  <slot>menuVSYNC()</slot>
  <slot>menuConfigureController()</slot>
  <slot>menuOpenMemCard()</slot>
  <slot>menuAbout()</slot>
  <slot>menuConfigureLogging()</slot>
  <slot>menuOpenUMD()</slot>
  <slot>menuOpenDebugger()</slot>
  <slot>menuAlwaysOnTop()</slot>
  <slot>menuSettings()</slot>
  <slot>menuScreenshot()</slot>
  <slot>menuUseShaders()</slot>
  <slot>menuMuteSound()</slot>
  <slot>menuFullscreen()</slot>
 </slots>
</ui>
